pages/dkp/konvoy/1.8/storage/static-local-volume/index.md

```diff
@@ -8,65 +8,49 @@ beta: false
 
-- Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
+-   Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
 
-# Before you begin
+## Before you begin
 
 Before starting this tutorial, you should verify the following:
 
-- You have access to a Linux, macOS, or Windows computer with a supported operating system version.
-
-- You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not any addons to the cluster.
+-   You have access to a Linux, macOS, or Windows computer with a supported operating system version.
 
-  For this tutorial, you **should not deploy** using all of the default settings as described in the [Quick start][quickstart].
+-   You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not any addons to the cluster.
 
-  This distinction between provisioning and deployment is important because some addons might depend on the storage class provided by the `localvolumeprovisioner` addon and might fail to start if it isn't configured yet.
+For this tutorial, you **should not deploy** using all of the default settings as described in the [Quick start][quickstart].
 
-## Provision the cluster and a volume
+This distinction between provisioning and deployment is important because some addons might depend on the storage class provided by the `localvolumeprovisioner` addon and might fail to start if it is not configured yet.
 
-1. Provision the Kubernetes cluster without deploying addons by running the following command:
+### Provision the cluster and a volume
 
-    ```bash
-    konvoy provision
-    ```
-
-1. Provision the local volume provisioner to watch for mounts in `/mnt/disks` on each host.
+1.  Provision the local volume provisioner to watch for mounts in `/mnt/disks` on each host.
 
     For example, mount a `tmpfs` volume with `ansible` using the `inventory.yaml` provided by `konvoy` by running the following command:
 
     ```bash
-    ansible -i inventory.yaml node -m shell -a "mkdir -p /mnt/disks/example-volume && mount -t tmpfs example-volume /mnt/disks/example-volume"
+    ANSIBLE_HOST_KEY_CHECKING=False ansible -i inventory.yaml node -m shell -b -a "mkdir -p /mnt/disks/example-volume && mount -t tmpfs example-volume /mnt/disks/example-volume"
     ```
 
-1. Deploy the `konvoy` cluster with addons by running the following command:
+1.  Verify the persistent volume by running the following command:
 
     ```bash
-    konvoy up
-    ```
-
-    When you run this command, the local volume provisioned detects the `example-volume` volume and adds it as a persistent volume to the `localvolumeprovisioner` storage class.
-
-1. Verify the persistent volume by running the following command:
-
-    ```bash
-    kubectl get pv
+    kubectl get pv --kubeconfig=admin.conf
```

Comment:
This PR looks good - but my one suggestion might be something like - 
"Verify the persistent volume by running the following command. Note: if you've already set your kubeconfig for this environment, ignore the `--kubeconfig=admin.conf` flag:
kubectl get pv --kubeconfig=admin.conf
"

Because some folks set that or might have it set for some other reason.

Other than that - nice improvements.